### PR TITLE
Readme file

### PR DESCRIPTION
**Pull request by ashish1codiot**
Readme file

### CODE REVIEW

**Comment by ashish1codiot** [comment]
this is only my review

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # My Project
-
-This is a project with the following dependencies:
-
+<details open>
+<summary>This is a project with the following dependencies:</summary>
+  
 - [accesscontrol](https://www.npmjs.com/package/accesscontrol): 2.2.1 (Latest: 2.2.1) [npm](https://www.npmjs.com/package/accesscontrol)
```
Comment (codewithashishdev) [inline README.md:5]: this is not working #1

File: README.md [hunk 2]
```diff
@@ -40,3 +40,5 @@ This is a project with the following dependencies:
 - [twilio](https://www.npmjs.com/package/twilio): 4.11.1 (Latest: 4.19.3) [npm](https://www.npmjs.com/package/twilio)
```
Comment (codewithashishdev) [inline README.md:40]: review #2

File: README.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 # My Project
-
-This is a project with the following dependencies:
-
+<details open>
+<summary>This is a project with the following dependencies:</summary>
+  
 - [accesscontrol](https://www.npmjs.com/package/accesscontrol): 2.2.1 (Latest: 2.2.1) [npm](https://www.npmjs.com/package/accesscontrol)
 - [bcrypt](https://www.npmjs.com/package/bcrypt): 5.1.0 (Latest: 5.1.1) [npm](https://www.npmjs.com/package/bcrypt)
 - [body-parser](https://www.npmjs.com/package/body-parser): 1.20.2 (Latest: 1.20.2) [npm](https://www.npmjs.com/package/body-parser)
```
Comment (codewithashishdev) [inline README.md:7]: REVIEV #3

**Review by codewithashishdev — APPROVED** [review]

**Pull request merged**